src/components/callToActionButton/CallToActionButton.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { expect, vi } from "vitest";
+import CallToActionButton from "./CallToActionButton.tsx";
+import { fireEvent, render } from "@testing-library/react";
+import { theme } from "../../theme/theme.ts";
+import { ThemeProvider } from "styled-components";
+
+test("CTA button onClick event calls function when button is clicked", () => {
+  const onClickMock = vi.fn();
+  const { getByText } = render(
+    <ThemeProvider theme={theme}>
+      <CallToActionButton handleClick={onClickMock}>
+        click me
+      </CallToActionButton>
+    </ThemeProvider>,
+  );
+
+  fireEvent.click(getByText("click me"));
+  expect(onClickMock).toHaveBeenCalledTimes(1);
```

Comment:
that's up to the team convention, but FYI we tend to put a new line for each expression that doesn't fit into a block. In a React component, it looks like this, e.g.:

react hooks block
{new line}
react query hooks
{new line}
custom hooks
{new line}
event handlers
{new line}

and so on - we're putting similar things visually together and split the blocks with a new line.

In tests it works the same

render
{new line}
constants
{new line}
event
{new line}
expect


and so on. Also, you can use a new line whenever your eyes have a hard time reading expressions that belong to the same group.